File: 2nd Semester/OOP Lab/Lab Manual 1/Home Tasks/Task7.java
import java.util.Scanner;

public class Task7 {// Integer Array Sorting.//
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] intArr = new int[10];
        int temp = 0;
        System.out.print("Enter Elements: ");
        for (int i = 0; i < 10; i++) {
            intArr[i] = scanner.nextInt();
        }
        scanner.close();
        for (int i = 0; i < 10; i++) {
            for (int j = i + 1; j < 10; j++) {
                if (intArr[i] > intArr[j]) {
                    temp = intArr[i];
                    intArr[i] = intArr[j];
                    intArr[j] = temp;
                }
            }
        }
        System.out.print("\nElements of array sorted in ascending order: ");
        for (int i = 0; i < 10; i++) {
            System.out.print(intArr[i] + " ");
        }
        System.out.print("\nElements of array sorted in decending order: ");
        for (int i = 9; i >= 0; i--) {
            System.out.print(intArr[i] + " ");
        }
    }
}

File: 1st Semester/Programming Fundamentals/Lab Work/Lab Manual 5/Lab Tasks/Task7.cpp
#include <iostream>
using namespace std;
int main()
{
   const int n=10;
    int i, j = 1, x, k;
    cout << "Enter the size of array: ";


    int array[n];
    for (i = 0; i < n; i++){
        cout << "Enter array element " << j << ": ";
        cin >> array[i];
        j++;
    }
    for (i = 0; i < n; i++){
        for (k = i + 1; k < n; k++){
            if (array[i] > array[k]){
                x = array[i];
                array[i] = array[k];
                array[k] = x;
            }
        }
    }
    cout << "Sorted array is\n";
    for (i = 0; i < n; i++){
        cout << array[i] << " ";
    }
    return 0;
}